google-cloud-pubsub/lib/google/cloud/pubsub/project.rb

```
@@ -133,14 +119,13 @@ def self.default_project
         #   pubsub = Google::Cloud::Pubsub.new
         #   topic = pubsub.topic "another-topic", skip_lookup: true
         #
-        def topic topic_name, autocreate: nil, project: nil, skip_lookup: nil
+        def topic topic_name, project: nil, skip_lookup: nil
           ensure_service!
           options = { project: project }
           return Topic.new_lazy(topic_name, service, options) if skip_lookup
           grpc = service.get_topic topic_name
           Topic.from_grpc grpc, service
         rescue Google::Cloud::NotFoundError
```

This now seems to amount to silently swallowing the exception. `NotFoundError` should be actually raised here, yes?